examples/ex07_save_load_model.nim

```diff
@@ -0,0 +1,93 @@
+import arraymancer, strformat, os
```

Comment:
You need to import it using relative imports, see the other examples.

Reply:
resolved